Libraries/Lists/ListView/__tests__/ListView-tests.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+'use strict';
+
+jest.disableAutomock();
+
+const React = require('React');
+const ReactTestRenderer = require('react-test-renderer');
+const ListView = require('ListView');
+const Text = require('Text');
+
+describe('ListView', () => {
```

Comment:
i would rather add tests to scrollview for this. listiview is delegating to scrollview#scrollTo